doc/classes/MainLoop.xml

```diff
@@ -140,5 +140,9 @@
 		<constant name="NOTIFICATION_TEXT_SERVER_CHANGED" value="2018">
 			Notification received when text server is changed.
 		</constant>
+		<constant name="NOTIFICATION_INPUT_ASYNC_MOUSE_MODE_UPDATED" value="2019">
+			Notification received from the OS when the [Input] mouse mode changed.
+			[b]Note:[/b] See [method DisplayServer.mouse_is_mode_async].
```

Comment:
This (and the Node.xml docs) should clarify that this notification only fires if `mouse_is_mode_async` is actually returning `true`. Might also make sense to refer to `mouse_set_mode` here. For example:

```suggestion
			Notification received from the OS after a request to asynchronously change the [Input] mouse mode succeeded.
			[b]Note:[/b] See [method DisplayServer.mouse_is_mode_async] and [method DisplayServer.mouse_set_mode].
```

New wording puts emphasis on "asynchronously change".